### ISSUE TITLE
Incompatible with ViteJS

### ISSUE THREAD

**Issue by rue-kcarlton** [kuasha420/mst-persistent-store#8]
I installed version 3.1.0 of this library for a React web app that was created with Vite. React projects created with Vite don't recognize the `process` global variable, so when trying to use in development for the first time, I get this error:`Uncaught ReferenceError: process is not defined`
versions:
```
    "localforage": "1.10.0",
    "mobx": "6.13.2",
    "mobx-react-lite": "4.0.7",
    "mobx-state-tree": "6.0.1",
    "mst-persistent-store": "3.1.0",
    "react": "^18.3.1",
    "vite": "^5.4.1"
```

**Comment by kuasha420** [comment]
Hi @rue-kcarlton :wave: Thanks for raising and sorry for the oversight. This has been fixed in `v3.1.1`. Give it a try and let me know if it is working in your setup.

**Comment by rue-kcarlton** [comment]
@kuasha420 - working better now, thank you.  Thank you for the quick turnaround!
